releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,12 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v7.0.0 - November, 20 2023
+**Breaking changes**
+  - Updated to fail task when it encounters multiline secrets.
```

Comment:
Updated to fail the task if any Key Vault references cannot be resolved or if the resolved values are invalid to be set as the pipeline secret.

Reply:
Updated

Reply:
Updated to fail **the** task if any Key Vault references cannot be resolved or if the resolved values are invalid to be set as the pipeline secret.

Reply:
Updated